File: easy/popsicles/Popsicles.c
#include <stdio.h>

int main() {
    int siblings;
    int popsicles;
    scanf("%d", &siblings);
    scanf("%d", &popsicles);
    
    printf("%s", popsicles % siblings ? "eat them yourself" : "give away");
    
    return 0;
}

File: easy/popsicles/Popsicles.java
import java.util.Scanner;

public class Popsicles {
    public static void main(String[] args) {
        var scanner = new Scanner(System.in);
        var siblings = scanner.nextInt();
        var popsicles = scanner.nextInt();
        
        var isEvenAmount = popsicles % siblings == 0;
        
        System.out.println(
            isEvenAmount ? "give away" : "eat them yourself"
        );
    }
}
